app/src/main/res/values-it/strings.xml

```diff
@@ -0,0 +1,263 @@
+        <item>0</item>
+        <item>1</item>
+    </string-array>
+    <string name="pref_name_autoSyncQueue_enabled">Sincronizzazione automatica separata per le modifiche locali</string>
+    <string name="pref_desc_autoSyncQueue_enabled">Questa opzione permette alle modifiche locali (link aggiunti, articoli archiviati, etc.) di essere sincronizzate sul server non appena venga resa disponibile una connessione.</string>
+    <string name="pref_name_autoDlNew_enabled">Scarica nuovi articoli all\'aggiunta</string>
+    <string name="pref_desc_autoDlNew_enabled">Scarica automaticamente nuovi articoli subito dopo essere stati aggiunti tramite l\'app. Nota: il download non ci sarà se avete modifiche offline non sincronizzate.</string>
+    <string name="pref_name_imageCache_enabled">Salva in cache le immagini degli articoli</string>
+    <string name="pref_desc_imageCache_enabled">Scarica automaticamente tutte le immagini in ogni articoli e le salva sul dispositivo. A seconda della lunghezza della tua lista di articoli di wallabag questo potrebbe richiedere molto tempo e consumare molto spazio.</string>
+
+    <string name="pref_categoryName_miscellaneous">Varie</string>
+    <string name="pref_name_misc_handleHttpScheme">Gestisci schema HTTP</string>
+    <string name="pref_desc_misc_handleHttpScheme">Mostra wallabag nella lista dei browser</string>
+    <string name="pref_name_misc_wipeDB">Cancella Database</string>
+    <string name="pref_desc_misc_wipeDB">Cancella tutti gli articoli dal database locale, rimuove anche tutte le modifiche locali e gli URL non sincronizzati</string>
+    <string name="pref_name_misc_wipeDB_confirmTitle">Cancella Database?</string>
+    <string name="pref_name_misc_wipeDB_confirmMessage">Sei sicuro di voler cancellare il database?</string>
+
+    <string name="connectionWizard_misc_incorrectConnectionURI">Usato URI di connessione sbagliato</string>
+
+    <string name="connectionWizard_welcome_titleMessage">Benvenuto in wallabag!</string>
+    <string name="connectionWizard_welcome_text">Prima di poter usare l\'app di wallabag devi configurare una connessione. Questo assistente di connessione cercherà di aiutarti.</string>
+    <string name="connectionWizard_welcome_rerunNotice">Puoi comunque riavviare l\'assistente di connessione dalle Impostazioni.</string>
+    <string name="connectionWizard_welcome_prev">Salta</string>
+    <string name="connectionWizard_welcome_next">Successivo</string>
+
+    <string name="connectionWizard_providerSelection_titleMessage">Seleziona il provider di wallabag</string>
+    <string name="connectionWizard_providerSelection_provider_general_name">Altro</string>
+    <string name="connectionWizard_providerSelection_provider_general_desc">Seleziona questa opzione per inserire le impostazioni richieste manualmente</string>
+    <string name="connectionWizard_providerSelection_provider_wallabagit_name" translatable="true">wallabag.it</string>
```

Comment:
This line should be removed (`translatable="false"` lines should be present only in the default locale).